common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -236,14 +175,14 @@ $link-color: rgb(26, 161, 222) !default;
           }
 
           &::after {
-            background: #333;
+            @include transform(rotate(45deg));
+            @include right(18px);
```

Comment:
Ah, yes, this can't be RTLed because the tooltips don't support RTL. Will revert.